### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN024.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#70] [General]
# https://blog.3place.cn/3place/lmyn/LMYN024.html

https://blog.3place.cn/3place/lmyn/LMYN024.html

**Comment by picklecai** [comment]
他就带了他们两个，航行在流向大海的河中。 （1.23.4）
到了大河的中流，
罗摩问牟尼魁首：“河水被劈开了，为什么这样怒吼？” （1.23.5）

没有理解罗摩问的是什么问题，对什么好奇。
众友的回答是“这是母亲河”。

**Comment by picklecai** [comment]
罗摩好像唐僧，不同的是唐僧哀叹如此可怕怎么得过，而少年罗摩，就只是问如此可怕到底是哪里。

众友：导游+武术老师+心理老师+历史老师+地理老师

**Comment by picklecai** [comment]
众友对两个孩子的称呼：
人中英豪！
制服敌人者！

说话时突然插进来，不仔细看会觉得怎么突然冒出这两词来？

**Comment by picklecai** [comment]
千眼大神因陀罗：

因陀罗酒后垂涎仙人乔达摩妻子的美貌。他为了得到乔达摩的妻子，便抓住月神让其帮忙。月神将日夜颠倒，因陀罗趁半夜时分化身为公鸡，躲在乔达摩屋外高声啼鸣，乔达摩听到鸡鸣之后，以为天亮，急匆匆赶出去做晨祷。这时，色心骤起的因陀罗趁虚而入，化身为乔达摩，摸进她的家里强行霸占了他的妻子。
待乔达摩回家后，知晓此事勃然大怒，但也于事无补，乔达摩发誓诅咒因陀罗，让其身上被打上1000个女人私处的印记。后来这1000个印记变成了1000只眼睛，这也是因陀罗为什么被称之为千眼之神的原因，但这也是他的耻辱记号。

**Comment by picklecai** [comment]
聪明的孙陀，孙陀的老婆母夜叉陀吒迦，生了个罗刹儿子摩哩遮。
这是罗刹版的《吉祥三宝》。

**Comment by picklecai** [comment]
离开这条路的半由旬，
她现在就住在那里，
我们走路必须经过，
陀吒迦的林子所在地。 （1.23.27）

你要靠你双臂的力量，
一定杀死这个坏东西，
你要按照我的命令，
清除这地方的荆棘。 （1.23.28）

罗摩的第一个任务来了。
